ROS/Chapter_5.ipynb

```diff
@@ -0,0 +1,505 @@
+{
```

Comment:
<p>reorder like we do in pymc resources?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/Bambi_resources/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>